packages/shared/src/utils/filePaths.test.ts

```diff
@@ -217,4 +218,35 @@ describe('truncateFileName', () => {
     expect(truncatedLength).toEqual(100);
     expect(truncated).toMatch(new RegExp('^this-title-.*-a-file-system-[a-z0-9]{4}$'));
   });
+  describe('removeLocalhostFromBaseUrl', () => {
```

Comment:
This test file has more test cases covered when the href for base element is any of these following:
- `http://localhost:3000/some/path/`
- `http://localhost:3000/some/path/?query=value#fragment`
- `/some/path/`
- `https://www.example.com/`